phpunit.xml.dist

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./tests/bootstrap.php" colors="true">
+<phpunit bootstrap="tests/bootstrap.php" colors="true">
     <testsuites>
         <testsuite name="Nekland Woketo">
-            <directory suffix="Test.php">./tests</directory>
```

Comment:
The suffix was great as there are many other files in the test folder, isn't it ?